core/isolate.rs

```diff
@@ -109,6 +110,16 @@ unsafe impl Send for Isolate {}
 
 impl Drop for Isolate {
   fn drop(&mut self) {
+    let zero_copy_bufs = self
+      .pending_ops
+      .iter_mut()
+      .filter(|op| !op.zero_copy_alloc_ptr.is_null())
+      .map(|op| replace(&mut op.zero_copy_alloc_ptr, null_mut()))
```

Comment:
I don't understand what's happening here - is this setting `op.zero_copy_alloc_ptr` to null?